### PR TITLE
PR for elastic search connection setup and tests

### PR DESCRIPTION
**Pull request by AnshPaktolus**

### CODE REVIEW

File: tests/test_data/test_config/test_elasticsearch.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+core_lib:
+  elasticsearch:
+    _target_: core_lib.connection.elasticsearch_connection_registry.ElasticSearchConnectionRegistry
+    config:
+      url:
+        protocol: http
+        host: ${oc.env:ELASTIC_HOST}
```
Comment (shubham-shay-te) [inline tests/test_data/test_config/test_elasticsearch.yaml:7]: let's add a test string over here and remove the environment variables for the test config

**Review by shay-te — APPROVED** [review]
after Shubham comment you can merge

**Review by shay-te — COMMENTED** [review]
https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=134&id=shay-te_core-lib&open=AYqOKHXNJXLSKmSuzJqz

File: core_lib/connection/elasticsearch_connection.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from core_lib.connection.connection import Connection
```
Comment (shay-te) [inline core_lib/connection/elasticsearch_connection.py:1]: https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=134&id=shay-te_core-lib&open=AYqOKHXNJXLSKmSuzJqz

**Review by shay-te — APPROVED** [review]

**Pull request merged**